meeshkan/serve/mock/faker/faker_base.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from faker import Faker
+from http_types import Request
+from openapi_typed import OpenAPIObject
+from openapi_typed_2 import Operation
+
+from meeshkan.serve.mock.storage import Storage
+
+
+class MeeshkanFakerBase:
+    def __init__(self, text_faker: Faker, request: Request, spec: OpenAPIObject, storage: Storage):
```

Comment:
Why's this called a `text_faker`?

Reply:
Because it fakes strings mostly. I wanted to separate our fakers from the 3rd party faker to not be confused.